src/main/java/health/HealthList.java

```diff
@@ -295,7 +296,7 @@ public static void printBmiHistory() throws CustomExceptions.OutOfBounds {
         int index = 1;
         System.out.println(HealthConstant.BMI_HISTORY_HEADER);
         for (Bmi bmi : BMIS) {
-            System.out.print(index + ". ");
+            System.out.print(index + UiConstant.INDEX);
```

Comment:
Use `UiConstant.SPLIT_BY_PERIOD` instead and a `UiConstant.SPLIT_BY_WHITESPACE` . Naming it index can be confusing.

Reply:
Correction: It was renamed to `FULL_STOP` so do take a look again.